common/src/main/java/com/wynntils/models/containers/ContainerModel.java

```diff
@@ -69,6 +78,9 @@ public final class ContainerModel extends Model {
     @Persisted
     private final Storage<Integer> finalMiscBucketPage = new Storage<>(10);
 
+    @Persisted
+    private final Storage<Map<Integer, List<ItemStack>>> bank = new Storage<>(new HashMap<>(21));
```

Comment:
@kristofbolyai this storage doesn't keep the data when closing the game even though I'm calling persist(). What am I doing wrong?

Reply:
Firstly we only allow storage treemaps, secondly whatever you want to do with predefined sizes wont work

Reply:
the predefined size doesn't really matter. how should I store the personal bank contents instead? Is the nested list ok? do I just have to change the HashMap to a TreeMap?